code/game/objects/structures/crates_lockers/closets/crittercrate.dm

```diff
@@ -34,6 +34,9 @@
 			if(prob(50))
 				content_mob = /mob/living/simple_animal/cat/Proc
 			new content_mob(loc)
+		else if(content_mob == /mob/living/simple_animal/butterfly)

```

Comment:
at this point you could probably just make the crate contain the animals anyway, as I think the list for unsendable objects is only for station->centcom and not the other way around.